### PR TITLE
[Sample] Create CI for sample service

### PR DESCRIPTION
**Pull request by D-W-**
This pull request includes various updates to documentation files and workflow files for improved organization and triggering conditions. The most important changes include renaming and updating the content of a readme file, updating triggering conditions for multiple workflow files, and adding a new workflow file for running daily and on pull requests.

Documentation updates:

* <a href="diffhunk://#diff-16d6365d8923a3a8b67448ce0fef412f6c0e48304129f9fc929420c498f5b44cL3-R21">`examples/tutorials/flow-deploy/create-service-with-flow/README.md`</a>: Renamed `readme.md` to `README.md` and updated its content to include instructions on starting the service and testing it with a request.
* <a href="diffhunk://#diff-49aaa2819e35a856818ecec8c9fa7e1c79ad028d3f44bd749736353cfb51bac9R31">`examples/README.md`</a>: Added a new entry for the `create-service-with-flow` example, including a badge for the workflow and a brief description.

Workflow updates:

* <a href="diffhunk://#diff-fb7c6e9c6f259fe228974b302120eae8d6e05762f555a0fcf364c02de1dbdcb8R1-R96">`.github/workflows/samples_tutorials_flow_deploy_create_service_with_flow.yml`</a>: Added a new workflow file that runs daily and on pull requests, with various steps including generating config files, setting up environments, running scripts, and uploading artifacts.
* <a href="diffhunk://#diff-a44df53d3ec63f6fb804dc7861f8591528dcbe8b2a136fd91c2fa8fa958575f8L12-R12">`.github/workflows/samples_tutorials_flow_deploy_kubernetes.yml`</a>: Updated triggering conditions to include the `examples/flows/integrations/**` path on pull requests.
* <a href="diffhunk://#diff-f7766a5c1b9b58a22fdeac882a0c1cc705d0041af20fd65b2ca1147a5babdc76L12-R12">`.github/workflows/samples_tutorials_flow_deploy_distribute_flow_as_executable_app.yml`</a>: Updated workflow file to remove the exclusion of the `examples/flows/integrations/**` path from triggering the workflow on pull requests.
* <a href="diffhunk://#diff-3d8d0c1a308ee42c43bc7fb8aa811e2ace33a28370c5fe83e0d2ed5029f0e96eL12-R12">`.github/workflows/samples_tutorials_e2e_development_chat_with_pdf.yml`</a>: Updated triggering conditions to remove the exclusion of the `examples/flows/integrations/**` path from triggering the workflow on pull requests.
* <a href="diffhunk://#diff-e33d15fd8335c327e5c2bd0c64edf8b9d6ba74d71b46cdcd847b4b46df5de9daL12-R12">`.github/workflows/samples_configuration.yml`</a>: Updated triggering conditions to remove the exclusion of `examples/flows/integrations/**` and add an exclusion for files ending with `requirements.txt`.
* <a href="diffhunk://#diff-2b597aa0e3ee9c2204912a626cc178ec1936a833fb52e40b38e2f0a8014ad2a5L12-R12">`.github/workflows/samples_tutorials_flow_deploy_azure_app_service.yml`</a>: Updated triggering conditions to remove the exclusion of the `examples/flows/integrations/**` path.
* <a href="diffhunk://#diff-f4d7aa3a413922191317263ca61fa5037e19395db3bd5c5993a8976878c1606dL12-R12">`.github/workflows/samples_tutorials_flow_deploy_docker.yml`</a>: Updated triggering conditions to remove the exclusion of the `examples/flows/integrations/**` path from triggering the workflow on pull requests.
* <a href="diffhunk://#diff-843ab2b8f6c9bf0a7e795425bd0e414d8bf60d320829a2f42a0329dab8ac140eL12-R12">`.github/workflows/samples_tutorials_flow_fine_tuning_evaluation_promptflow_quality_improvement.yml`</a>: Updated triggering conditions to remove the exclusion of the `examples/flows/integrations/**` path from triggering the workflow on pull requests.# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: .github/workflows/samples_configuration.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ on:
     - cron: "40 22 * * *" # Every day starting at 6:40 BJT
   pull_request:
     branches: [ main ]
-    paths: [ examples/**, examples/*requirements.txt, .github/workflows/samples_configuration.yml, '!examples/flows/integrations/**' ]
+    paths: [ examples/**, examples/*requirements.txt, .github/workflows/samples_configuration.yml ]
```
Comment (zhengfeiwang) [inline .github/workflows/samples_configuration.yml:12]: please reverted such changes to exclude integration examples

Comment (D-W-) [reply .github/workflows/samples_configuration.yml:12]: Created a bug to track. [Bug 2839343](https://msdata.visualstudio.com/Vienna/_workitems/edit/2839343): [Promptflow SDK& CLI] Fix readme CI template

File: examples/tutorials/flow-deploy/create-service-with-flow/README.md [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
+```bash
+nohup python simple_score.py & 
+sleep 10
+```
+
+## 2. Test the service with request
+
+```bash
+curl -X POST http://127.0.0.1:5000/score --header "Content-Type: application/json" --data '{"flow_input": "some_flow_input", "node_input": "some_node_input"}'
 ```
```
Comment (crazygao) [inline examples/tutorials/flow-deploy/create-service-with-flow/README.md:23]: explain the purpose of simple score.
add some sample output.

File: examples/tutorials/flow-deploy/create-service-with-flow/README.md [hunk 3]
```diff
@@ -1,12 +1,23 @@
...
 You can use json parser to parse the output.
-For example, in powershell:
 
-```powershell
-curl -X POST http://127.0.0.1:5000/score --header "Content-Type: application/json" --data '{"flow_input": "some_flow_input", "node_input": "some_node_input"}' | ConvertFrom-Json | ConvertTo-Json
+## 1. Start the service and put in background
+
+```bash
+nohup python simple_score.py & 
+sleep 10
```
Comment (wangchao1230) [inline examples/tutorials/flow-deploy/create-service-with-flow/README.md:16]: comment that this is not needed for user, but for ci

**Review by crazygao — APPROVED** [review]

**Review by 0mza987 — APPROVED** [review]

**Review by wangchao1230 — APPROVED** [review]

**Pull request merged**